openshift/openshift_transport.go

```diff
@@ -57,7 +57,7 @@ func ParseReference(ref string) (types.ImageReference, error) {
 	}
 	tagged, ok := r.(reference.NamedTagged)
 	if !ok {
-		return nil, errors.Errorf("invalid image reference %s, %#v", ref, r)
+		return nil, errors.Errorf("invalid image reference %s, %#v, expected format: 'hostname/namespace/stream:tag'", ref, r)
```

Comment:
(`a/b/c/host/foo` is not intended to be valid, just a random test string I was using primarily for the other message)